metrics/prometheus/prometheus.go

```diff
@@ -429,6 +431,19 @@ func NewMetrics(cfg config.PrometheusMetrics, disabledMetrics config.DisabledMet
 			"Seconds from when the connection was requested until it is either created or reused",
 			[]string{adapterLabel},
 			standardTimeBuckets)
+
+		if !metrics.metricsDisabled.AdapterConnectionDialMetrics {
+			metrics.adapterConnectionDials = newCounter(cfg, reg,
+				"adapter_connection_dials",
+				"Count number of started dials to open connections with adapter bidder endpoints.",
+				[]string{adapterLabel})
+
+			metrics.adapterConnectionDialTime = newHistogramVec(cfg, reg,
+				"adapter_connection_dial_time_seconds",
```

Comment:
No need to include "seconds" suffix. This is the default on prometheus. You can check, but I don't think we do this for the other time metrics / histograms.